### PR TITLE
Lightpoints

### LINKED ISSUE #453: Lightpoints and createVehicle
### Expected behaviour

Creating a local lightpoint to the player will save a lot of fps drops for all the players.

### Actual behaviour

Since copLights.sqf and medicLights.sqf are remoteExecuted for all players, those scripts create a lightpoint which isn't local that could lead to fps drops since you will have 2 * playersOnline lightpoints created when activating the lights.

### Steps to reproduce the behaviour

Turn on the medic/cop lights a lot of times in less than 2/3 seconds and all the players will have some fps drops (with a large amount of players, like 60+).

My actual fix:

From:

_lightleft = "#lightpoint" createVehicle getPos _vehicle;

To:

_lightleft = "#lightpoint" createVehicleLocal getPos _vehicle;

### [RPT](https://community.bistudio.com/wiki/Crash_Files#Arma_3) and/or extDB3 logs (if applicable)

----

Mission version: 
All versions
Launcher version: 
1.82
Game version: 

Branch:

### PR DESCRIPTION
**Pull request by xAndreei**
Resolves #453 . 

#### Changes proposed in this pull request: 
- From createVehicle to createVehicleLocal.

- [x] I have tested my changes and corrected any errors found
Changes have been tested and works perfectly.

### CODE REVIEW

File: Altis_Life.Altis/core/medical/fn_medicLights.sqf [hunk 1]
```diff
@@ -2,80 +2,73 @@
...
+        _leftRed = true;  
+        _lightleft setLightBrightness 0.0;  
         sleep 0.05;
-        _lightright setLightBrightness 6;
-    };
-    sleep (_this select 1);
-};
+        _lightright setLightBrightness 6;  
+    };  
+    sleep (_this select 1);
```
Comment (BoGuu) [inline Altis_Life.Altis/core/medical/fn_medicLights.sqf]: We can also moves this over to `params`

File: Altis_Life.Altis/core/cop/fn_copLights.sqf [hunk 2]
```diff
@@ -2,130 +2,104 @@
...
 
-//ARE YOU ALL HAPPY?!?!?!?!?!?!?!?!?%#?@WGD?TGD?BN?ZDHBFD?GA
 _lightleft setLightDayLight true;
 _lightright setLightDayLight true;
 
-_leftRed = true;
-while{ (alive _vehicle)} do
-{
+_leftRed = true;  
+while { (alive _vehicle) } do
```
Comment (BoGuu) [inline Altis_Life.Altis/core/cop/fn_copLights.sqf]: Make sure to keep in line with the general style (same line brace, as you've done in the other file. Can remove the parenthesis here (and whitespace).

File: Altis_Life.Altis/core/cop/fn_copLights.sqf [hunk 3]
```diff
@@ -2,130 +2,104 @@
...
+        _leftRed = true;  
+        _lightleft setLightBrightness 0.0;  
         sleep 0.05;
-        _lightright setLightBrightness 6;
-    };
-    sleep (_this select 1);
-};
+        _lightright setLightBrightness 6;  
+    };  
+    sleep (_this select 1);
```
Comment (BoGuu) [inline Altis_Life.Altis/core/cop/fn_copLights.sqf]: Ditto

File: Altis_Life.Altis/core/cop/fn_copLights.sqf [hunk 4]
```diff
@@ -2,130 +2,104 @@
...
-{
+_leftRed = true;  
+while { (alive _vehicle) } do  
+{  
     if (!(_vehicle getVariable "lights")) exitWith {};
-    if (_leftRed) then
-    {
-        _leftRed = false;
-        _lightright setLightBrightness 0.0;
+    if (_leftRed) then   {
```
Comment (BoGuu) [inline Altis_Life.Altis/core/cop/fn_copLights.sqf]: Double check this whitespacing

**Comment by Jason2605** [comment]
Thanks for the PR!

**Pull request merged**